app/src/main/java/com/example/bigowlapp/utils/GeoLocationFormatter.java

```diff
@@ -10,7 +10,7 @@
 
 
 public class GeoLocationFormatter {
-    public static String formatLocation(Context context, GeoPoint geoPoint) {
+    public String formatLocation(Context context, GeoPoint geoPoint) {
```

Comment:
add empty list check in the case the location formatter gets 0 results. Right now the app crashes in that case.